### DISCUSSION TITLE
image classification

### DISCUSSION THREAD

**Discussion by faalkao** [apache/incubator-mxnet#19394] [Q&A] [answered]
Dear All,
I am doing on image classifications, namely land cover classification with satellite images of multiple channels/bands. Can anyone suggest good deep learning with code?
Thank you in advance.

**Comment by szha** [comment]
GluonCV offers plenty of model options for image classification: https://gluon-cv.mxnet.io/model_zoo/classification.html

In addition, @feevos and team have done several great work that are related satellite imaging who likely have more insight to share.

**Comment by feevos** [comment, marked as answer]
Dear @faalkao, for land cover classification (segmentation) you can take a look at our work on semantic segmentation and change detection. Repositories with mxnet <2.0 can be found on https://github.com/feevos/resuneta and https://github.com/feevos/ceecnet. The accompanying papers are mentioned in the repositories, except [this one](https://www.sciencedirect.com/science/article/pii/S0034425720301115), that is on field boundaries delineation and the deep learning model used was ResUNet-a. Happy to help in any questions you may have. 

All the best,
Foivos
